<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sur</class>
 <widget class="QStackedWidget" name="sur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="currentIndex">
   <number>2</number>
  </property>
  <widget class="QWidget" name="page_sur_view_edit">
   <widget class="QPushButton" name="page_sur_view_edit_button_2d_zoom_in">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_3d_hide">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hide 3D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_3d_fullscreen">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fullscreen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="page_sur_view_edit_insert_annotations">
    <property name="geometry">
     <rect>
      <x>423</x>
      <y>390</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Insert annotation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_2d_fullscreen">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fullscreen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_logout">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_2d_zoom_out">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom Out</string>
    </property>
   </widget>
   <widget class="QOpenGLWidget" name="page_sur_view_edit_2d_view">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>341</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="page_sur_view_edit_toolbar_help">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;TOOLBAR, NOT SURE HOW TO IMPLEMENT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QOpenGLWidget" name="page_sur_view_edit_3d_view">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_preview_report">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>634</x>
      <y>530</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview Report</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="page_sur_view_edit_scroll">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>16</width>
      <height>341</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_2d_previous_note">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous Note</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_2d_previous_slice">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous Slice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_2d_next_note">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Note</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_2d_next_slice">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Nexts Slice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_2d_play_paus">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play/Paus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_3d_left">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_3d_right">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_3d_up">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_3d_down">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_3d_zoom_in">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_3d_zoom_out">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_3d_tissue_view">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Tissue View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_3d_fullscreen_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Full Screen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_3d_bone_view">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Bone View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_view_edit_button_3d_toggle_annotations">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Toogle Annotations</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_sur_patient_errand">
   <widget class="QPushButton" name="page_sur_patient_errand_button_back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_patient_errand_button_logout">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="page_sur_patient_errand_rad_report">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>301</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="page_sur_patient_errand_errand_list">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>371</width>
      <height>391</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Task</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Scan</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Clinic</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>1</string>
     </property>
     <property name="text">
      <string>01-01-2020</string>
     </property>
     <property name="text">
      <string>MRI</string>
     </property>
     <property name="text">
      <string>US</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="page_sur_patient_errand_search_box">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Search&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_patient_errand_button_share">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Share</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_patient_errand_button_download">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_patient_errand_button_view">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_sur_home">
   <widget class="QPushButton" name="page_sur_home_logout">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_home_button_my_profile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>My profile </string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="page_sur_home_treeWidget_treatment_list">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>461</width>
      <height>451</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Patient ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date ordered</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Order ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>0000</string>
     </property>
     <property name="text">
      <string>01-01-2020</string>
     </property>
     <property name="text">
      <string>9999</string>
     </property>
     <property name="text">
      <string>Pending</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0000</string>
     </property>
     <property name="text">
      <string>02-02-2020</string>
     </property>
     <property name="text">
      <string>3333</string>
     </property>
     <property name="text">
      <string>Complete</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="page_sur_home_textEdit_search">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Search box&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_sur_home_button_proceed">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Proceed</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
